components/prism-powershell.js

```diff
@@ -9,26 +9,38 @@ Prism.languages.powershell = {
 			lookbehind: true
 		}
 	],
-	'string': {
-		pattern: /("|')(`?[\w\W])*?\1/m,
-		inside: {}
-	},
+	'string': [
+		{
+			pattern: /"(`?[\w\W])*?"/m,
+			inside: {
+				'function': {
+					pattern: /[^`]\$\(.*?\)/,
+					inside: {}
```

Comment:
Could you add a comment like "See at the end" or something, so that this looks more intentional?